test/integration/scheduler/scheduler_test.go

```diff
@@ -714,4 +578,325 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			framework.ExpectWorkloadsToBeFrozen(ctx, k8sClient, fooCq.Name, wl2)
 		})
 	})
+
+	ginkgo.When("Using taints and affinity in resourceFlavors", func() {
```

Comment:
we can split this to taints and affinity since they don't share cq nor flavors

Reply:
done :)